src/ImageSharp/Formats/Png/PngDecoderCore.cs

```diff
@@ -1590,18 +1561,17 @@ private PngChunkType ReadChunkType()
             {
                 return (PngChunkType)BinaryPrimitives.ReadUInt32BigEndian(this.buffer);
             }
-            else
-            {
-                PngThrowHelper.ThrowInvalidChunkType();
 
-                // The IDE cannot detect the throw here.
-                return default;
-            }
+            PngThrowHelper.ThrowInvalidChunkType();
+
+            // The IDE cannot detect the throw here.
+            return default;
```

Comment:
Do we still need this with the `DoesNotReturn` attribute?

Reply:
It shouldn’t (I don’t think) but I was still getting a warning 🤔

Maybe we need to turn on Nullable Reference Types in a future PR? (Which I would love)

Reply:
Would also love to have that enabled. Probably going to be another major PR 😉